### PR TITLE
Add CIP for calling procedures

### PR DESCRIPTION
**Pull request by boggle**

### CODE REVIEW

File: cip/CIP2015-06-24-call-procedures.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
+= CIP2015-06-24 Calling Stored Procedures
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Rickard Öberg <rickard.öberg@neotechnology.com>
```
Comment (Mats-SX) [inline cip/CIP2015-06-24-call-procedures.adoc]: I guess this email is no longer active?

File: cip/CIP2015-06-24-call-procedures.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+== Proposal
+
+This proposal consists of two parts, introducing stored procedures conceptually, and describing how to call them in a
+system that supports Cypher.
+
+=== Stored procedures
+
+A stored procedure has a name and takes a fixed number of named and typed arguments in a fixed sequence and either just
+yields nothing (i.e. causes a side effect) or yields a stream of result records.  Each record yielded contains values for a fixed number of named and typed columns in the same fixed sequence such that each column value in a yielded record
+matches its column's type.
```
Comment (Mats-SX) [inline cip/CIP2015-06-24-call-procedures.adoc]: Weird newline.

**Comment by Mats-SX** [comment]
Most of the paragraphs in this document do not adhere to the 'one sentence per line' style guide.

**Comment by boggle** [comment]
Will re-open after an overhaul of this.

**Pull request closed without merging**